contracts/SimpleListFactory.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.24;
+
+import './SimpleList.sol';
+
+contract SimpleListFactory {
+  mapping (address => address) public tokenToListMap;
+
+  event SimpleListCreated(address list, address admin);
+
+  function createSimpleList(address token) public returns (address) {
```

Comment:
not sure it's best cause that way not issuers can "steal" cc addresses
@LiorRabin @bejavu